atm.js

```diff
@@ -1,21 +1,68 @@
-var checkingBalance = document.querySelector("div#checking_balance");
-checkingBalance = parseInt( checkingBalance.innerHTML.replace("$", "") );
 
-var checkingDeposit = function(amount){
-  checkingBalance = parseInt( checkingBalance.innerHTML.replace("$", "") );
+$(document).ready (function(){
```

Comment:
This is not wrong, but `document.ready` is unecessary here because your script tag is just before the closing body tag. the DOM will have loaded by the time your script is loaded